StyleGuide.sql

```diff
@@ -0,0 +1,73 @@
+-- # General
+
+-- Never mention database names in a query because they may not be the same across our platforms
+
+-- Capitalise keywords and function names
+-- Qualify table names with schema name
+-- Each field and part of statement should be on its own line
+-- Indent for readability
+UPDATE
+	[schema].[Table]
+SET
+	[FieldOne] = 'Thing',
+	[FieldTwo] = GETDATE()

```

Comment:
comma should be at the beginning of next line, causes smaller diffs when new fields are added (1 line changed compared to 2)

Reply:
Done.